pkg/services/correlations/api.go

```diff
@@ -298,8 +298,20 @@ type GetCorrelationsBySourceUIDResponse struct {
 // 404: notFoundError
 // 500: internalServerError
 func (s *CorrelationsService) getCorrelationsHandler(c *contextmodel.ReqContext) response.Response {
+	perPage := c.QueryInt64("perpage")
```

Comment:
I think it other APIs the param is called "limit", so it'd be good to keep it consistent.